projection/src/main/scala/akka/projection/r2dbc/internal/R2dbcProjectionImpl.scala

```diff
@@ -316,14 +316,16 @@ private[projection] object R2dbcProjectionImpl {
       offsetStore: R2dbcOffsetStore)(implicit
       ec: ExecutionContext,
       system: ActorSystem[_]): FlowWithContext[Envelope, ProjectionContext, Done, ProjectionContext, _] = {
+    val warnAboutFilteredEvents =
+      system.settings.config.getBoolean("akka.projection.r2dbc.warn-about-filtered-events-in-flow")
```

Comment:
I'd prefer to place it in R2dbcProjectionSettings and pass in to adaptedHandlerForFlow. I see now that I was lazy and never changed it from `case class`. Fortunately the whole R2dbcProjection is still `@ApiMayChange` so we should cleanup that.

Reply:
Turned normal class in [e7c2fee](https://github.com/akka/akka-persistence-r2dbc/pull/339/commits/e7c2feec867f13b75b94658c54f1926bb5268b92)